recipes/numberStoryTime.tsx

```diff
@@ -0,0 +1,164 @@
+import { h } from "@commontools/html";
+import {
+  derive,
+  handler,
+  ifElse,
+  JSONSchema,
+  lift,
+  llm,
+  NAME,
+  recipe,
+  schema,
+  str,
+  UI,
+} from "@commontools/builder";
+
+const inputSchema = schema({
+  type: "object",
+  properties: {
+    number: { type: "number", default: 0 },
+  },
+}) satisfies JSONSchema;
+
+const nextNumber = handler<{}, { number: number }>(
+  (_, state) => {
+    state.number = (state.number ?? 0) + 1;
+  },
+);
+const randomNumber = handler<{}, { number: number }>(
+  (_, state) => {
+    state.number = Math.floor(Math.random() * 102) - 1;
+  },
+);
+
+const generateStory = lift(({ number }: { number: number }) => {
+  return {
+    system:
+      "You are the parent of a young child who loves to learn about numbers. Luckily for your child, you are a historian of numbers and when the child says a number you make up an interesting story about it, including the history of the number.",
+    messages: [`Tell me a story about ${number}`],
```

Comment:
you could use

```
str`Tell me a story about ${number}`
```

here and not make it a `lift`. just in case.

Reply:
heh, I forgot about this